### PR TITLE
Redis user session

### PR DESCRIPTION
**Pull request by aaomidi**

### CODE REVIEW

File: redis/redis.go [hunk 1]
```diff
@@ -137,3 +140,19 @@ func GetVersionUploadState(versionID string) (*generated.CreateVersionResponse,
...
+	redisKey := "user:token:" + token + ":revoked"
+
+	client.Set(redisKey, true, time.Hour*24*30)
+}
+
+func IsAccessTokenRevoked(token string) bool {
+	redisKey := "user:token:" + token + ":revoked"
+	res, err := client.Exists(redisKey).Result()
+	if err != nil {
+		log.Err(errors.New("redis call to get user token failed"))
```
Comment (aaomidi) [inline redis/redis.go:154]: I think I might need a Msg here?

File: redis/redis.go [hunk 2]
```diff
@@ -38,7 +40,8 @@ func InitializeRedis(ctx context.Context) {
 		panic(ping.Err())
 	}
 
-	log.Ctx(ctx).Info().Msg("Redis initialized")
+	log = zl.Ctx(ctx)
+	log.Info().Msg("Redis initialized")
```
Comment (Vilsol) [inline redis/redis.go:44]: Log should be accessed from context, not a global variable

File: redis/redis.go (hunk 1 above)
Comment (Vilsol) [inline redis/redis.go:154]: Should get logger from context

**Review by Vilsol — CHANGES_REQUESTED** [review]

File: redis/redis.go (hunk 1 above)
Comment (Vilsol) [reply redis/redis.go:154]: Yes, otherwise nothing will get printed

**Pull request closed without merging**